### PR TITLE
Add resolution-related terms and definitions

### LINKED ISSUE #125: [NTR] Add resolution definitions
There are four types of data resolution: spatial, spectral, radiometric and temporal. Many instruments can capture one or two types of resolution simultaneously, yet it is rare to find equipment capable of delivering on all four types. This phenomenon is known as the resolution trade-off. Most instruments measure the most commonly used types of resolutions: spatial and spectral. Together, spatial and spectral resolutions allow scientists to quantitatively measure factors such as color, space and detail.

## Preferred term labels and textual definitions
* resolution =def: A [physical quality](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001018) of a measurement device that expresses (realizes) the device's capacity to distinguish between two distinct but closely related measurements of the same physical quality or type, often quantified as the smallest detectable difference (resolution interval) between measured values. 
* resolution interval =def: A [data item](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000027) that quantifies a resolution, by representing the minimum measurable difference between two measurements of the same physical quality or type, such that the measurements can be distinguished as different.
   * spatial resolution interval =def: A resolution interval that quantifies the minimum distance between two points such that they can be considered two separate points in a given microscope under specific experimental parameters.
      * lateral resolution interval as angle =def: A spatial resolution interval that quantifies the minimum angular separation between two points on a plane orthogonal to the light ray, expressed as an angle centered in the lens (Formula: δθ = 1.22 * λ / D).
      * lateral resolution interval as length =def: A spatial resolution interval that quantifies the minimum linear separation between two points on a plane orthogonal to the light ray, expressed as the distance from the center of the optical spot (Formula: δr = 0.61 * λ / NA).
      * axial resolution interval as length =def: A spatial resolution interval that quantifies the minimum measurable separation between two points along the axis of propagation of the light ray, expressed as a linear distance (Formula: δz = 2 ⋅ n ⋅ λ / NA²).
      * diffraction limit =def: A spatial resolution interval that quantifies the theoretical maximum spatial resolution achievable by an optical system, constrained by the diffraction of light.
   * spectral resolution interval =def: A resolution interval that quantifies the minimum separation between two narrow peaks in a spectrum such that they can be distinguished as two separate peaks.
      * spectral resolution interval as wavelength =def: A spectral resolution interval that quantifies the minimum separation between two wavelengths such that they can be distinguished as different (Formula: δλ = λ / (n * N)).
      * spectral resolution interval as wavenumber =def:  A spectral resolution interval that quantifies the minimum separation between two wavenumbers such that they can be distinguished as different (Formula: δν~ = δλ / λ²).
      * spectral resolution interval as frequency =def:  A spectral resolution interval that quantifies the minimum separation between two energy levels such that they can be distinguished as different (Formula: δE = h * δν = h * c * δν~ = h * c * δλ / λ²).
      * spectral resolution interval as energy =def:  Spectral resolution expressed as difference of energies. δE
   * radiometric resolution interval =def: A resolution interval that quantifies the minimum difference in irradiated energy, often coded as gray level in a grayscale or look-up-table (LUT).
   * temporal resolution =def: A resolution interval that quantifies the temporal separation between two measurements or frames, often expressed as the inverse of the frame rate. In techniques where temporal resolution is not directly derived from frame rate, it quantifies the minimum temporal interval between two resolvable events.   
* resolution power =def: A [ratio](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FSTATO_0000184) that quantifies the ability of a measurement device to distinguish between small differences in measurement values, expressed as the ratio between a typical measurement datum and the corresponding resolution interval.

## Synonyms

...

## Textual definition

s. above

## Link to [CHARISMA Wiki equivalent](https://wiki.charisma.ideaconsult.net/wiki/List_of_all_terms)

- https://wiki.charisma.ideaconsult.net/wiki/Item:Q22 Resolution (Raman)  
- https://wiki.charisma.ideaconsult.net/wiki/Item:Q200 Spectral resolution  
- https://wiki.charisma.ideaconsult.net/wiki/Item:Q198 Pixel resolution 

None of them is exhausting.

## Suggested parent term

- resolution - [PATO:physical quality](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001018)
- resolution interval - [IAO:data item](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000027)
- resolution power - [STATO:ratio](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FSTATO_0000184) 

## Examples

Please provide a more detailed description that examplifies how the term is supposed to be used, in terms of what it is suppose to repesent (e.g. a concept or a relation).

## Attribution

If you would like a nanoattribution, please indicate your ORCID id

## Sources
* https://www.hunterlab.com/blog/what-is-spectral-resolution/ --> https://en.wikipedia.org/wiki/Image_resolution#Types  
* https://phys.libretexts.org/Bookshelves/Mathematical_Physics_and_Pedagogy/Computational_Physics_(Chong)/11%3A_Discrete_Fourier_Transforms/11.02%3A_Spectral_Resolution_and_Range
* https://en.wikipedia.org/wiki/Optical_resolution
* http://vikdhillon.staff.shef.ac.uk/teaching/phy217/instruments/phy217_inst_dispersion.html
* https://www.azom.com/article.aspx?ArticleID=13369 https://www.ossila.com/pages/spectrometer-resolution

### PR DESCRIPTION
**Pull request by Zack-83**
Closed #125

### CODE REVIEW

**Review by StroemPhi — APPROVED** [review]
Again many thanks!

File: src/templates/vibso_classes.tsv [hunk 1]
```diff
@@ -28,6 +28,20 @@ obo:VIBSO_0000024	point distance Z	sampling pace in the Z direction|sampling ste
...
+obo:VIBSO_0000038	lateral resolution interval as angle		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	Spatial resolution on a plane orthogonal to the light ray, expressed as an angle centered in the lens.	, ISBN 081946043-5	owl:Class	spatial resolution interval	ORCID: 0000-0003-2673-366X|ORCID:0000-0002-2239-3955		$δθ = 1.22\cdotλ/D$				
+obo:VIBSO_0000039	lateral resolution interval as length		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	Spatial resolution on a plane orthogonal to the light ray, expressed as dittance ftom the centre of the spot.	, ISBN 081946043-5	owl:Class	spatial resolution interval	ORCID: 0000-0003-2673-366X|ORCID:0000-0002-2239-3955		$δr = 0.61\cdotλ/NA$ (Abbe formula, derived from the Rayleigh criterion with the Kirchhoff, Debye and paraxial approximation) $= 1.22 λ\cdot(f/\sharp)$
```
Comment (Zack-83) [inline src/templates/vibso_classes.tsv:42]:
```suggestion
obo:VIBSO_0000039	lateral resolution interval as length		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	Spatial resolution on a plane orthogonal to the light ray, expressed as distance from the centre of the spot.	, ISBN 081946043-5	owl:Class	spatial resolution interval	ORCID: 0000-0003-2673-366X|ORCID:0000-0002-2239-3955		$δr = 0.61\cdotλ/NA$ (Abbe formula, derived from the Rayleigh criterion with the Kirchhoff, Debye and paraxial approximation) $= 1.22 λ\cdot(f/\sharp)$				
```

**Comment by StroemPhi** [comment]
@Zack-83 I've linked to the terms in the CHARISMA wiki that you provided in the issue, apart from the [Pixel resolution ](https://wiki.charisma.ideaconsult.net/wiki/Item:Q198), since we didn't define it here. Should we, or can we ignore it? 
If we should also define it, then I propose opening a new PR for it, so we can merge this one.

**Comment by Zack-83** [comment]
> @Zack-83 I've linked to the terms in the CHARISMA wiki that you provided in the issue, apart from the [Pixel resolution ](https://wiki.charisma.ideaconsult.net/wiki/Item:Q198), since we didn't define it here. Should we, or can we ignore it? If we should also define it, then I propose opening a new PR for it, so we can merge this one.

Clearly the definition of "image resolution" (~ NUMBER of distinct neighbouring measurements) is not coherent with the definition of "optical resolution" (~ DISTANCE between neighbouring measurement values). Therefore I agree with you to leave this term out.

**Pull request merged**